service/core/kafka/server/src/main/scala/com/lightbend/lagom/internal/broker/kafka/TopicProducerActor.scala

```diff
@@ -4,52 +4,46 @@
 
 package com.lightbend.lagom.internal.broker.kafka
 
-import akka.kafka.ProducerSettings
-import akka.kafka.scaladsl.{ Producer => ReactiveProducer }
-import akka.stream.scaladsl.GraphDSL
+import akka.actor.Actor
+import akka.actor.ActorLogging
+import akka.actor.Props
 import akka.actor.Status
-import akka.stream.scaladsl.Source
+import akka.kafka.scaladsl.DiscoverySupport
+import akka.kafka.scaladsl.{ Producer => ReactiveProducer }
```

Comment:
It might be time to drop the import rename.

Reply:
It's impossible because this package has a `Producer` class too 🤷‍♂️